datalad/distribution/clone.py

```diff
@@ -221,7 +221,7 @@ def __call__(
         # combine all given sources (incl. alternatives), maintain order
         for s in [source] + assure_list(alt_sources):
             candidate_sources.extend(_get_flexible_source_candidates(s))
-        lgr.info("Cloning dataset from '%s' (trying %i location candidates) to '%s'",
+        lgr.info("Cloning dataset from '%s' (trying %i location candidate(s)) to '%s'",
                  source, len(candidate_sources), dest_path)
```

Comment:
As stated in the summary above the test is an open issue, but your's passes. If we have time, I will eventually come up with a leaner test. We know that master is broken, so that doesn't require one....